<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiezoWidget</class>
 <widget class="QWidget" name="PiezoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Ax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="xDisplay_a_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xGround_a_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xChange_a_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Ay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="yDisplay_a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yGround_a">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yChange_a">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
   </widget>
   <zorder>yGround_a</zorder>
   <zorder>yChange_a</zorder>
   <zorder>yDisplay_a</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_Az">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="zDisplay_a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zGround_a">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zChange_a">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>zGround_a</zorder>
   <zorder>zChange_a</zorder>
   <zorder>zDisplay_a</zorder>
  </widget>
  <widget class="QGroupBox" name="comPortGroupBoxA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>COM Port</string>
   </property>
   <widget class="QComboBox" name="comPortComboA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="connectLabel_a">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A: not connected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Bx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="xDisplay_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xGround_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xChange_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>xGround_b</zorder>
   <zorder>xChange_b</zorder>
   <zorder>xDisplay_b</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_By">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="yDisplay_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yGround_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yChange_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>yGround_b</zorder>
   <zorder>yChange_b</zorder>
   <zorder>yDisplay_b</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_Bz">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="zDisplay_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z: 0 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zGround_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zChange_b">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>zGround_b</zorder>
   <zorder>zChange_b</zorder>
   <zorder>zDisplay_b</zorder>
  </widget>
  <widget class="QLabel" name="connectLabel_b">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string> B: not connected</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="comPortGroupBoxB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>COM Port</string>
   </property>
   <widget class="QComboBox" name="comPortComboB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="connectB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
